Jenkinsfile

```diff
@@ -1,12 +1,14 @@
 pipeline {
     agent none
-
+    environment {
+        UID = buildHelper.getUid()
```

Comment:
This method won’t work with agents, since there’s no point getting the master UID when the container is then run on an agent. It *must* be the uid where the container is run no matter what.

Reply:
I have no idea how to solve this then

Reply:
Actually, It might be ok with the templates we’re using, since the uid is fixed at 498, so assuming that is consistent this would be fine , but maybe with a todo to note that it wouldn’t work outside the specific templates.

Reply:
All good, @akashisama found the solution, just putting up now

Reply:
Actually it might not need to be dynamic... that templates are fixed at 498, so we might get away with setting that as the default value rather than trying to calculate it.

Reply:
You can run ``additionalBuildArgs '--build-arg BUILDER_UID=$(id -u)'`` and it will dynamically set as appropriate